xcode/Subconscious/Shared/Components/Common/Tabs/TabHeaderView.swift

```diff
@@ -47,7 +47,6 @@ struct TabHeaderView: View {
                     .frame(width: increment - inset, height: 3)
                     .cornerRadius(2, corners: [.topLeft, .topRight])
                     .offset(x: inset / 2 + increment * CGFloat(focusedTabIndex), y: 0)
-                    .opacity(0.5)
```

Comment:
The little purple bar under the profile tabs, I think it looks better at full opacity.